<launch>
    <node name="Pipe_Sim_Map"       pkg="pipe_sim"      type="pipe_map_node"      output="screen">
    <param name="name"              type="string"       value="PipeMap"/>
    </node>
    <node name="Robot"              pkg="pipe_sim"      type="robot_node"      output="screen">
    <param name="count"             type="int"          value="1000"/> 
    <param name="distance"          type="double"       value="9.0"/>
    <param name="velocity"          type="double"       value="0.0"/>
    <param name="direction"         type="bool"         value="true"/>
    <param name="start_noise"       type="double"       value="5.0"/>
    <param name="move_noise"        type="double"       value="0.2"/>
    <param name="sense_noise"       type="double"       value="1.0"/>
    <param name="refresh_rate"      type="double"       value="10.0"/>
    <param name="sense_rate"        type="double"       value="0.2"/>
    <param name="sense_range"       type="double"       value="0.5"/>
    <param name="simulated_robot"   type="bool"         value="true"/>
    </node>
</launch>
